pygmt/tests/test_plot.py

```diff
@@ -231,6 +220,29 @@ def test_plot_colors_sizes_proj(data, region):
     return fig
 
 
+@pytest.mark.mpl_image_compare
+def test_plot_varying_intensity():
+    """
+    Plot the data with array-like intensity.
+    """
+    x = np.arange(1, 10)
+    y = np.arange(1, 10)
+    intensity = np.linspace(-1, 1, len(x))
+
+    fig = Figure()
+    fig.plot(
+        x=x,
+        y=y,
+        region=[0, 10, 0, 10],
+        projection="X10c",
+        frame=True,
+        style="c0.5c",
+        color="blue",
+        intensity=intensity,
+    )
+    return fig
```

Comment:
What about rewriting the test like this? In the new test, the x values denote the intensity of circles. I feel that it's easier to verify if the baseline image is correct.
```
import numpy as np
import pygmt

x = np.arange(-1, 1.1, 0.1)
y = np.zeros(x.size)
intensity = x 

fig = pygmt.Figure()
fig.plot(
    x=x,
    y=y,
    region=[-1.1, 1.1, -0.5, 0.5],
    projection="X15c/2c",
    frame=["S", "xaf+lIntensity"],
    style="c0.5c",
    color="blue",
    intensity=intensity,
)
fig.show()
```
![image](https://user-images.githubusercontent.com/3974108/111925760-937fb800-8a80-11eb-996c-db6d63092b13.png)

Reply:
Done in b3b7734.